### PR TITLE
Add views and controllers for Patients, Physicians, Appointments and Diagnosis items.

### PR DESCRIPTION
**Pull request by YuliiaManivska**
Created controllers and  views.

### CODE REVIEW

File: lesson5/app/controllers/appointments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  def destroy
+    @appointment = Appointment.find(params[:id])
+    @appointment.destroy
+    redirect_to appointments_path
+  end
+
+  private def appointment_params
+    params.require(:appointment).permit(:physician_id, :patient_id, :appointment_date)
+  end
```
Comment (olehvavryniv) [inline lesson5/app/controllers/appointments_controller.rb]:
```suggestion
  private 

  def appointment_params
    params.require(:appointment).permit(:physician_id, :patient_id, :appointment_date)
  end
```

File: lesson5/app/helpers/appointments_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AppointmentsHelper
+end
```
Comment (olehvavryniv) [inline lesson5/app/helpers/appointments_helper.rb]: Please remove

File: lesson5/app/controllers/appointments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  def index
+    @appointment = Appointment.all
+  end
+
+  def show
+    @appointment = Appointment.find(params[:id])
+  end
+
+  def new
+    @appointment = Appointment.new
```
Comment (olehvavryniv) [inline lesson5/app/controllers/appointments_controller.rb:11]:
```suggestion
    @appointment = Appointment.new
    @patients = Patients.all
```

File: lesson5/app/views/appointments/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<div class = "row m-4">
+  <h2>Edit appointment</h2>
+  <%= form_for :appointment, url: appointment_path(@appointment), method: :patch do |f| %>
```
Comment (olehvavryniv) [inline lesson5/app/views/appointments/edit.html.erb]:
```suggestion
  <%= form_for @appointment do |f| %>

```

File: lesson5/app/views/appointments/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<div class = "row justify-content-center m-3">
```
Comment (olehvavryniv) [inline lesson5/app/views/appointments/index.html.erb]:
```suggestion
<div class="row justify-content-center m-3">

```

File: lesson5/app/views/appointments/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,28 @@
+<div class ="row justify-content-center">
+  <div class = "col-5">
+    <div class = "card">
+      <div class = "card-header">
+      Appointment № <%= @appointment.id %>
+      </div>
+      <div class = "card-body">
+        <p>
+        Physician: 
+        <%= @appointment.physician.first_name%>
```
Comment (olehvavryniv) [inline lesson5/app/views/appointments/show.html.erb]:
```suggestion
        <%= @appointment.physician.first_name %>

```

File: lesson5/app/assets/stylesheets/appointments.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Appointments controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YuliiaManivska) [inline lesson5/app/assets/stylesheets/appointments.scss:3]: Remove

**Review by olehvavryniv — APPROVED** [review]

**Pull request merged**